src/Compilers/Core/Portable/PEWriter/Members.cs

```diff
@@ -22,7 +22,6 @@ namespace Microsoft.Cci
     /// <summary>
     /// Specifies how the caller passes parameters to the callee and who cleans up the stack.
     /// </summary>
-    // PROTOTYPE(func-ptr): represent `unmanaged`
```

Comment:
>// PROTOTYPE(func-ptr): represent `unmanaged` [](start = 4, length = 45)

Is this what ```Standard``` already means?

Reply:
No, `Standard` is the `StdCall` convention. `unamanged` is supposed to represent whatever the platform-default convention is. `StdCall` is an unfortunate name for the convention, but it's also what everyone calls it so we're stuck with it.

---
In reply to: [409826087](https://github.com/dotnet/roslyn/pull/43355#discussion_r409826087) [](ancestors = 409826087)

Reply:
> No, Standard is the StdCall convention. unamanged is supposed to represent whatever the platform-default convention is. StdCall is an unfortunate name for the convention, but it's also what everyone calls it so we're stuck with it.

I used Standard in term of this enum. 
So, is ```unmanaged``` already represented in this enum or not?

---
In reply to: [409839891](https://github.com/dotnet/roslyn/pull/43355#discussion_r409839891) [](ancestors = 409839891,409826087)

Reply:
No, the addition is tracked in [https://github.com/dotnet/roslyn/issues/39865](https://github.com/dotnet/roslyn/issues/39865).

---
In reply to: [409850766](https://github.com/dotnet/roslyn/pull/43355#discussion_r409850766) [](ancestors = 409850766,409839891,409826087)